src/Altinn.AccessManagement.Core/Helpers/DelegationHelper.cs

```diff
@@ -46,30 +44,30 @@ public static class DelegationHelper
         }
 
         /// <summary>
-        /// Gets a string representation of the CoveredByPartyId
+        /// Trys to get the PartyId attribute value from a list of AttributeMatch models
         /// </summary>
-        /// <returns>The CoveredByPartyId value</returns>
-        public static bool TryGetCoveredByPartyIdFromMatch(List<AttributeMatch> match, out int coveredByPartyId)
+        /// <returns>The party id if found as the single attribute in the collection</returns>
+        public static bool TryGetPartyIdFromAttributeMatch(List<AttributeMatch> match, out int partyid)
         {
-            coveredByPartyId = 0;
+            partyid = 0;
             if (match?.Count == 1 && match.First().Id == AltinnXacmlConstants.MatchAttributeIdentifiers.PartyAttribute)
```

Comment:
Burde konstanten het noe med partyid siden vi bruker party begrepet nå for alle typer i url?

Reply:
The AltinnXacmlConstants were defined in altinn-authorization and have been copied into both access-management and resource-registry so a renaming should probably be syncronized between repos?

It's just the naming internally though, the actual urn value for the PartyAttribute and UserAttribute are:
```
urn:altinn:partyid
urn:altinn:userid
```